### PR TITLE
Use "ember-native-dom-helpers" for acceptance and integration tests

### PR DESCRIPTION
**Pull request by Turbo87**
see https://github.com/cibernox/ember-native-dom-helpers

Disclaimer: This is still not perfect and using jQuery in some cases. Eventually we should use e.g. `ember-test-selectors` to get rid of those jQuery selectors.

/cc @sivakumar-kailasam @mschinis @toddjordan @cibernox

### CODE REVIEW

File: tests/.eslintrc.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 module.exports = {
-  env: {
```
Comment (sivakumar-kailasam) [inline tests/.eslintrc.js]: was the removal of this block intentional?

Comment (Turbo87) [inline tests/.eslintrc.js]: yes, because those helpers (except for `currentURL()`) should not be used anymore when you use `ember-native-dom-helpers`

Comment (sivakumar-kailasam) [inline tests/.eslintrc.js]: 👍

**Comment by sivakumar-kailasam** [comment]
Will wait for @toddjordan to review this once before we can merge.

File: tests/.eslintrc.js (hunk 1 above)
Comment (cibernox) [inline tests/.eslintrc.js]: Actually `ember-native-dom-helpers` also has a currentURL. You can import it and use it.

Comment (Turbo87) [inline tests/.eslintrc.js]: @cibernox seems to be missing from https://github.com/cibernox/ember-native-dom-helpers#helpers

Comment (cibernox) [inline tests/.eslintrc.js]: @Turbo87 Fixed.

**Pull request merged**